diff --git a/dist/vue-dsfr.js b/dist/vue-dsfr.js
index 15c64cf6f42b6295bd92ed8195262d276bce21c7..60b27d2acfeba16846e255fc06682912b58edc7b 100644
--- a/dist/vue-dsfr.js
+++ b/dist/vue-dsfr.js
@@ -22,7 +22,7 @@ const st = Symbol("accordions"), rt = Symbol("header"), He = Symbol("tabs"), he
     adjust: s,
     onTransitionEnd: (r, c = !0) => {
       var u, v;
-      e.value = !1, c && ((v = (u = o.value) == null ? void 0 : u.querySelector("a")) == null || v.focus()), o.value && r === !1 && o.value.style.removeProperty("--collapse-max-height");
+      e.value = !1, c && ((v = (u = o.value) == null ? void 0 : u.querySelector("a")) == null), o.value && r === !1 && o.value.style.removeProperty("--collapse-max-height");
     }
   };
 }, ma = "(prefers-color-scheme: dark)", ga = "vue-dsfr-scheme";
@@ -3325,10 +3325,7 @@ const al = {
     }, i = b(() => t.isAlert ? "alertdialog" : "dialog"), r = V(null), c = V();
     ue(() => t.opened, (y) => {
       var H, q;
-      y ? ((H = c.value) == null || H.showModal(), setTimeout(() => {
-        var h;
-        (h = r.value) == null || h.focus();
-      }, 100)) : (q = c.value) == null || q.close(), u(y);
+      y ? ((H = c.value) == null || H.showModal()) : (q = c.value) == null || q.close(), u(y);
     });
     function u(y) {
       typeof window < "u" && document.body.classList.toggle("modal-open", y);
