diff --git a/dist/vue-dsfr.js b/dist/vue-dsfr.js
index e042994a90c3bfd7f4250b6d788207315d6e0060..817585b78f96827c03d4ac9548336a8f10dbe518 100644
--- a/dist/vue-dsfr.js
+++ b/dist/vue-dsfr.js
@@ -22,7 +22,7 @@ const rt = Symbol("accordions"), nt = Symbol("header"), Ke = Symbol("tabs"), ye
     adjust: s,
     onTransitionEnd: (r, f = !0) => {
       var d, v;
-      e.value = !1, f && ((v = (d = l.value) == null ? void 0 : d.querySelector("a")) == null || v.focus()), l.value && r === !1 && l.value.style.removeProperty("--collapse-max-height");
+      e.value = !1, f && ((v = (d = l.value) == null ? void 0 : d.querySelector("a")) == null), l.value && r === !1 && l.value.style.removeProperty("--collapse-max-height");
     }
   };
 }, ba = "(prefers-color-scheme: dark)", ha = "vue-dsfr-scheme";
@@ -3343,10 +3343,7 @@ const so = {
     }, o = b(() => t.isAlert ? "alertdialog" : "dialog"), r = R(null), f = R();
     de(() => t.opened, (k) => {
       var O, K;
-      k ? ((O = f.value) == null || O.showModal(), setTimeout(() => {
-        var y;
-        (y = r.value) == null || y.focus();
-      }, 100)) : (K = f.value) == null || K.close(), d(k);
+      k ? ((O = f.value) == null || O.showModal()) : (K = f.value) == null || K.close(), d(k);
     });
     function d(k) {
       typeof window < "u" && document.body.classList.toggle("modal-open", k);
